app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
```

Comment:
it`s unnecessary to leave pass

Reply:
you can also delete a comment

Reply:
@AnnaBoichenko
Sorry, but this red  area means that the rows were by default in the code.
Please, take a look to the green area.